### ISSUE TITLE
general problem with content for KeyboardHelpDialog

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#488]
There's a general problem with the content for `KeyboardHelpDialog`, demonstrated by `OhmsLawKeyboardHelpContent`:

```js
  function OhmsLawKeyboardHelpContent( tandem ) {

    var sliderControlsHelpContent = new SliderControlsHelpContent();
    var generalNavigationHelpContent = new GeneralNavigationHelpContent();

    var content = new HBox( {
      children: [ sliderControlsHelpContent, generalNavigationHelpContent ],
      align: 'top',
      spacing: 30
    } );

    Panel.call( this, content, {
      stroke: null,
      fill: 'rgb( 214, 237, 249 )',
      tandem: tandem
    } );
  }
```

(1) The Panel is unnecessary.

(2) By wrapping the content in a Panel, you're unintentionally adding Panel's default margins (xMargin:5, yMargin:5).   In `KeyboardHelpDialog`, `var HELP_CONTENT_MARGIN = 15;`.  Do you'll need to increase this to `20`.

(3) The Panel has copied and hardcoded `fill: 'rgb( 214, 237, 249 )'` from KeyboardHelpDialog. Change the fill in either place and you're going to get one weird looking dialog. 

Places where Panel should be removed (search for `Panel` in "*.HelpContent.js"):

- [x] BASEKeyboardHelpContent
- [x] GravityForceLabBasicsKeyboardHelpContent
- [x] JohnTravoltageKeyboardHelpContent
- [x] MoleculesAndLightKeyboardHelpContent
- [x] OhmsLawKeyboardHelpContent
- [x] PlinkoProbabilityKeyboardHelpContent
- [x] ResistanceInAWireKeyboardHelpContent

**Comment by pixelzoom** [comment]
I fixed all of these except for JohnTravoltageKeyboardHelpContent.  It's doing something evil at the end of its constructor:

```js
    // the content should be centered in the dialog relative to the description text
    var oldCenter = contentHBox.centerX;
    var newCenter = descriptionVBox.centerX;
    var spacerWidth = newCenter - oldCenter + 2 * DIALOG_MARGIN;
    contentHBox.addChild( new Spacer( spacerWidth, 0 ) );
```

Assigning to @jessegreenberg and @zepumph to handle `JohnTravoltageKeyboardHelpContent` and review.

**Comment by zepumph** [comment]
I removed the panel from JT. I decided to keep the code smell illustrated in https://github.com/phetsims/joist/issues/488#issuecomment-384403141. JT was the first keyboard navigation dialog, and the content is by no means idiomatic. When we publish this sim again, we will update the keyboard dialog content to a consistent look with `HelpContent.js`. I'm fine letting it slide until then. JT is not high priority right now, and would design time for the updated dialog.

@pixelzoom your commits look good; thank you for doing this.

**Comment by pixelzoom** [comment]
`JohnTravoltageKeyboardHelpContent ` cleanup move to https://github.com/phetsims/john-travoltage/issues/279.

Closing.
